content/events/2018/tech-day-02-17.md

```diff
@@ -3,15 +3,8 @@ title: February Tech Day 2
 date: 2018-02-17
 ---
 
-{{< location 25 >}}
+{{% location building-25 %}}
 
 The day will run from 10:00am until 5:00pm, however feel free to come and go as you please.
 
-## Food
-Food on campus is scarce on weekends, so we recommend either ordering in food, or bringing something with you. There are a few corner-shops located nearby however. There's a [map of local food places](https://goo.gl/yYlfs5) to help you find nearby food.
-
-## Electronics
-As expected, you'll need to bring your robot to work on, in whatever state it may be. We also recommend bringing at least 1 laptop with you to work on. Laptops will be provided for you to use.
```

Comment:
Can you double-check the last sentence? I think I was expecting it to contain a "not", though I don't know whether that's the case.

Reply:
You are correct, that last sentence should read _very_ differently!